contract/tools/e2e-tools.js

```diff
@@ -228,25 +230,37 @@ export const provisionSmartWallet = async (
   }
 
   /** @type {import('../test/wallet-tools.js').MockWallet['offers']} */
-  const offers = Far('Offers', {
-    executeOffer,
-    /** @param {string|number} offerId */
-    tryExit: offerId => sendAction({ method: 'tryExitOffer', offerId }),
-  });
+  const offers = makeExo(
+    'Offers',
+    M.interface('Offers', {}, { defaultGuards: 'passable', sloppy: true }),
+    {
+      executeOffer,
+      /** @param {string|number} offerId */
+      tryExit: offerId => sendAction({ method: 'tryExitOffer', offerId }),
+    },
+  );
 
   /** @type {import('../test/wallet-tools.js').MockWallet['deposit']} */
-  const deposit = Far('DepositFacet', {
-    receive: async payment => {
-      const brand = await E(payment).getAllegedBrand();
-      const asset = vbankEntries.find(([_denom, a]) => a.brand === brand);
-      if (!asset) throw Error(`unknown brand`);
-      /** @type {Issuer} */
-      const issuer = asset.issuer;
-      const amt = await E(issuer).getAmountOf(payment);
-      await sendFromWhale(asset.denom, amt.value);
-      return amt;
+  const deposit = makeExo(
+    'DepositFacet',
+    M.interface(
+      'DepositFacet',
+      {},
+      { defaultGuards: 'passable', sloppy: true },
+    ),
+    {
+      receive: async payment => {
+        const brand = await E(payment).getAllegedBrand();
```

Comment:
indenting everything further is tedious.

If we factor out the interface, can we avoid that?

```suggestion 
  const sloppyGuards = { defaultGuards: 'passable', sloppy: true };
  const depositI = M.interface('DepositFacet', {}, sloppyGuards);
  const deposit = makeExo('DepositFacet', depositI, {
    receive: async payment => {
      const brand = await E(payment).getAllegedBrand();
```